staging/src/k8s.io/client-go/INSTALL.md

```diff
@@ -83,10 +83,10 @@ go mod init
 ```
 
 Indicate which version of `client-go` your project requires.
-For `client-go` on master (and once version v12.0.0 is released), this is a single step:
+For `client-go` on v12.0.0 (and later), this is a single step:
 
 ```sh
-go get k8s.io/client-go@master # or v12.0.0+ once released
+go get k8s.io/client-go@v12.0.0
```

Comment:
do we have to bump the version every time a new patch is out for v12.0.0+?

Reply:
_Technically_, it doesn't matter because the pattern remains the same. But I'd prefer bumping the version so that installation instructions always reflect the current release.